docs/en/observability/create-alerts.asciidoc

```diff
@@ -7,10 +7,14 @@ Make sure to set up alerting in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
```

Comment:
"Make sure to set up alerting in Kibana" - this reads a little awkward.
Ensure Alerting is already set up in {kib}? Or "You should..." variant?

Reply:
Thanks. Yeah, "Ensure Alerting is already set up in {kib}" sounds better.